templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -168,3 +171,14 @@ async def chat_request(
         result=_Message(role=MessageRole.ASSISTANT, content=response.response),
         nodes=_SourceNodes.from_source_nodes(response.source_nodes),
     )
+
+
+@r.get("/config")
```

Comment:
We have a Swagger document so i think it's best to create a config pydantic model so that the user can understand what the output is (check the chat and message model above)